dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -24,16 +26,19 @@
 import { defineComponent, inject } from "vue";
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/e932fc87-e65b-4578-ad23-197d558b983c)
Question, would it be compicated/high effort to modify the close sign to look like in the figma?

Reply:
I am not sure. Will have a quick look at it.

Reply:
done

Reply:
thank you!